docs/requirements.md

```diff
@@ -0,0 +1,51 @@
+# BMC Functional and Quality Requirements
+
+
+### Version
+| Date  |  Author | Content  |
+|---|---|---|
+|  2022-02-08 | Andrew Nieuwsma  | Initial Draft  |
+
+
+## Overview
+
+This document describes the functional and quality requirements that CSM expects any Redfish based BMC (baseboard management controller) controlling nodes to support.  This document is intended to give objective measureable requirements that correspond with the [profiles](../profiles/) listed.
+
+Some of these requirements transcend BMC requirements and relate to the interconnection between BMC and nodes. 
+
+## Definitions:
+
+1. Redfish - Protocol developed by [DMTF](https://www.dmtf.org/standards/redfish)
+2. BMC - Baseboard Management Controller, hereafter referred to as the 'BMC', this entity provides the Redfish interfact and controls the node.
+3. Node - the hardware component that is controlled by the BMC, hereafter referred to as the 'node'.
+4. Customer - The Cray System Managment (CSM) services is the consumer of the redfish API hosted on BMCs.  The customer refers to CSM and its usage pertaining to the Redfish APIs on the BMC.
+
+
+## Functional Requirements
+
+|  ID  |  Requirement | Notes  |
+|---|---|---|
+|  FR1 | The BMC shall support the paths, parameters, and cabilities specified in the [profiles](../profiles/) listing.   Here is a restatement of some of the core support requirements: The BMC shall at a minimum support: <ol><li>Power actions of the node and the BMC</li><li>Firmware inventory/update of the node and the BMC</li><li>Redfish Alerts and Events</li><li>telemetry (either streammed (ideal) or polled) of the node</li><li>power capping of the node.</li></ol> | The Customer has specified a minimum set of Redfish API requirments needed to support its operation. |
```

Comment:
- streammed -> streamed
- Capitalize #4 (telemetry) and #5 (power capping) to match the others?